cloud-info/cloud-info-wrapper.sh

```diff
@@ -1,9 +1,15 @@
 #!/bin/sh
 
+EXTRA_OPTS=""
+
+if test "xDEBUG" = "x1"; then
```

Comment:
```suggestion
if test "x${DEBUG}" = "x1"; then
```

Reply:
well spotted, that explains why I was not getting the results I expected :)

Reply:
But honestly I always find this test quite ugly.
Usuall I prefer things relying on `-z` and `-n`.
So if you want to rely test the DEBUG value something like
```shell
if [ -n "$DEBUG" ] && [ "$DEBUG" = "1" ]; then
  EXTRA_OPTS="$EXTRA_OPTS --debug"
fi
```